apps/task/models.py

```diff
@@ -45,23 +45,52 @@ def __str__(self):
     @staticmethod
     def get_for(user):
         return TaskGroup.objects.filter(
-            user_group__members=user
+            users__in=[user]
         ).distinct()
 
+    def duration_taskgroup(self, user):
+        total_duration = Task.objects.order_by().filter(
+                    task_group=self.id,
+                    user=user
+                ).values('task_group').annotate(
+                    task_duration=F('timeentry__end_time') - F('timeentry__start_time')
+                ).aggregate(
+                    sum=Sum('task_duration')
+                )['sum']
+        return total_duration
+
 
 class Task(BaseModel):
     title = models.CharField(max_length=255)
     description = models.TextField(blank=True)
     external_url = models.URLField(max_length=255,
                                    blank=True, null=True)
     task_group = models.ForeignKey(TaskGroup, on_delete=models.CASCADE,
-                                    blank=True, null=True)
+                                   blank=True, null=True)
     user = models.ForeignKey(User, on_delete=models.CASCADE,
                              blank=True, null=True)
 
     def __str__(self):
         return self.title
 
+    @staticmethod
+    def get_for(user):
+        task_group = TaskGroup.get_for(user)
+        return Task.objects.filter(
+            user=user,
+            task_group__in=task_group
+        ).distinct()
```

Comment:
```suggestion
        return Task.objects.filter(
            user=user,
        ).distinct()
```
What do you think? We can only add user to task groups in which he is allowed?